docs/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml

```diff
@@ -162,6 +168,28 @@
             </row>
 
             <row>
+              <entry><emphasis>FOLD<indexterm>
+                  <primary>FOLD</primary>
```

Comment:
@richardkchapman I am going to need some more info about the big picture of the rights/signing enhancement. It seems to span several areas. 

Is this PR good for the time being? IOW, can the rest be added at the time signing is added?